packages/front-end/next.config.js

```diff
@@ -1,3 +1,32 @@
+const fs = require("fs");
+const path = require("path");
+
+const rootPath = path.join(__dirname, "..", "..");
+
+let gitSha = "";
+let gitCommitDate = "";
+if (fs.existsSync(path.join(rootPath, "buildinfo", "SHA"))) {
+  gitSha = fs
+    .readFileSync(path.join(rootPath, "buildinfo", "SHA"))
+    .toString()
+    .trim();
+}
+if (fs.existsSync(path.join(rootPath, "buildinfo", "DATE"))) {
+  gitCommitDate = fs
+    .readFileSync(path.join(rootPath, "buildinfo", "DATE"))
+    .toString()
+    .trim();
+}
+
+fs.writeFileSync(
+  path.join(__dirname, "styles", "variables.scss"),
+  `$public-asset-prefix: "${
+    process.env.USE_REMOTE_ASSETS
```

Comment:
The logic in `api/init.ts` is different:
```typescript
  const publicAssetsPath =
    IS_CLOUD && USE_REMOTE_ASSETS
      ? `https://growthbook-cloud-static-files.s3.amazonaws.com/${build.date}/${build.sha}/public`
      : "";
```
Is it possible to have only one reference implementation?